stdlib/ballerina-database/src/main/java/org/ballerinalang/database/sql/SQLDatasourceUtils.java

```diff
@@ -820,18 +804,64 @@ public static void setUserDefinedValue(Connection conn, PreparedStatement stmt,
         return new Object[] { structData, structuredSQLType };
     }
 
+    /**
+     * This will close the database connection.
+     *
+     * @param conn SQL connection
+     */
+    public static void cleanupResources(Connection conn, boolean isInTransaction) {
```

Comment:
shall we also define isInTransaction in java doc ?

Reply:
+1